server/src/uuid/model/taxonomy_term/mod.rs

```diff
@@ -553,4 +554,100 @@ impl TaxonomyTerm {
 
         Ok(taxonomy_term)
     }
+
+    pub async fn create_entity_link<'a, E>(
+        payload: &taxonomy_create_entity_link_mutation::Payload,
+        executor: E,
+    ) -> Result<(), operation::Error>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        let instance_id = Self::get_instance_id(payload.taxonomy_term_id, &mut transaction).await?;
+
+        for child_id in payload.entity_ids.clone() {
```

Comment:
```suggestion
        for child_id in payload.entity_ids {
```

Use `*child_id` instead of `child_id` then